docs/bis/ec/ec-format.md

```diff
@@ -86,6 +87,23 @@ Supported scientific type:
 - `normalized`
 - `zeroNormalized`
 
+**ratioType** determines the type of ratio format to use. Only valid and is required when the type is ratio.
```

Comment:
we need to add an example of a ratio format ... we should have added examples of bearing and azimuth formats as well ... @rschili can we add some examples?

Reply:
done